cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -1,74 +1,92 @@
+* Before this update, running the command `odo init --name <name> --devfile-path <path>` caused a panic to occur with a non-existent path. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5540[(odo-5540)]
 
-Provide as many details in the issue description as possible.
+* Before this update, `odo` commands ignored namespace settings in the `.odo/env/env.yaml` file. With this update, `odo` applies the namespace setting.
+link:https://github.com/redhat-developer/odo/issues/5544[(odo-5544)]
 
-.For Documentation
+* Before this update, running the `odo dev` command could fail after you deployed a component with `odo deploy`. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5546[(odo-5546)]
 
-If you find an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+* Before this update, the `odo dev` command did not use the `.gitignore` file when synchronizing files to a container. With this update, `odo` uses the `.gitignore` file.
+link:https://github.com/redhat-developer/odo/issues/5581[(odo-5581)]
 
+* Before this update, the `odo deploy` and `odo build-images` did not work on Windows. With this update, `odo` can now build images on Windows.
+link:https://github.com/redhat-developer/odo/issues/5588[(odo-5588)]
 
+* Before this update, the `odo delete` command failed for Kubernetes components. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5637[(odo-5637)]
 
+* With this update, `odo` now supports the `command` field in container components.
+link:https://github.com/redhat-developer/odo/issues/5648[(odo-5648)]
 
+* Before this update, when you stopped the `odo dev` process, resources created using `odo deploy` were removed. With this update, `odo` only deletes resources that were deployed using `odo dev`.
+link:https://github.com/redhat-developer/odo/issues/5700[(odo-5700)]
 
-////
-[id="odo-known-issues_{context}"]
-== Known issues
+* Before this update, `odo` would display volumes defined in the devfile as containers. This update fixes the issue.
+link:https://github.com/redhat-developer/odo/issues/5779[(odo-5779)]
 
-////
+* Before this update, `odo` would save the component name in the `env.yaml` file. With this update, the component name is only specified in the devfile.
+link:https://github.com/redhat-developer/odo/issues/5780[(odo-5780)]
 
-//[id="odo-technology-preview_{context}"]
-//== Technology Preview features `{odo-title}`
+
+* Before this update, after terminating the `odo dev` command, `odo` would delete the `ServiceBinding` resource but the logs would have an entry stating "Failed to delete the `ServiceBinding` resource". This update fixes the issue.
```

Comment:
```suggestion
* Before this update, after terminating the `odo dev` command, `odo` would delete the `ServiceBinding` resource, but the logs would have an entry stating `Failed to delete the ServiceBinding resource`. This update fixes the issue.
```
Per the ISG, elements like log entries should be in monospace (i.e. backticks) and not quotation marks